### PR TITLE
More symbol names 9

### PR DESCRIPTION
**Pull request by rocky**
Partial go over of graphics, importexport, options, operations, and plot,
 
More work could be done on graphics, and plot, but that is left for later. This is already large.
    
Also another pass over everything to improve specific calls is needed  - later.

<a href="https://gitpod.io/#https://github.com/Mathics3/mathics-core/pull/335"></a>

### CODE REVIEW

File: mathics/builtin/drawing/plot.py [hunk 1]
```diff
@@ -416,9 +420,10 @@ def apply(self, functions, x, start, stop, evaluation, options):
                 functions_param[index] = f
             functions = functions.flatten_with_respect_to_head(SymbolList)
 
-        expr_limits = Expression(SymbolList, x, start, stop)
+        expr_limits = ListExpression(x, start, stop)
+        # FIXME: arrange forself to have a .symbolname property or attribute
         expr = Expression(
-            self.get_name(), functions, expr_limits, *options_to_rules(options)
+            Symbol(self.get_name()), functions, expr_limits, *options_to_rules(options)
```
Comment (TiagoCavalcante) [inline mathics/builtin/drawing/plot.py:426]: Maybe `get_head` can be used here?

**Review by TiagoCavalcante — APPROVED** [review]
LGTM

File: mathics/builtin/drawing/plot.py (hunk 1 above)
Comment (rocky) [reply mathics/builtin/drawing/plot.py:426]:
```
(trepan3k) self.get_head
** AttributeError: 'Plot' object has no attribute 'get_head'
```

(Re)read the FIXME  that appears on line 424.

**Review by mmatera — APPROVED** [review]
LGTM

**Pull request merged**